src/Command.php

```diff
@@ -161,11 +161,50 @@ public function putItem($table, array $value, array $options = [])
      */
     public function batchPutItem($table, array $values, array $options = [])
     {
+        assert(count($values <= 25));
         list($name, $argument) = $this->db->getQueryBuilder()->batchPutItem($table, $values, $options);
         return $this->setCommand($name, $argument);
     }
     
     /**
+     * Put multiple items in the table with size > 25 object.
+     * @param string $table   The name of the table.
+     * @param array  $values  The values to input.
+     * @param array  $options Additional options to the request argument.
+     * @return static[]
+     */
+    public function batchPutAllItems($table, array $values, array $options = [])
+    {
+        $batches = array_chunk($values, 25);
+        $commandOriginal = $this;
+        return array_map(
+            function ($batch) use ($commandOriginal, $table, $options) {
+                list($name, $argument) = $this->db->getQueryBuilder()->batchPutItem($table, $batch, $options);
+                $command = clone $commandOriginal;
```

Comment:
$this will point to Command object? Sorry, my mind clouded with JavaScript where the $this points to the closure @_@